care/facility/migrations/0420_auto_20240312_2318.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.10 on 2024-03-12 17:48
+
+from django.db import migrations
+
+def double_pain_scale(apps, schema_editor):
+    DailyRound = apps.get_model('facility', 'DailyRound')
+    for daily_round in DailyRound.objects.all():
+        if daily_round.pain_scale_enhanced:
+            for item in daily_round.pain_scale_enhanced:
+                if 'scale' in item:
+                    item['scale'] *= 2
+            daily_round.save()
+
+def halve_pain_scale(apps, schema_editor):
+    DailyRound = apps.get_model('facility', 'DailyRound')
+    for daily_round in DailyRound.objects.all():
+        if daily_round.pain_scale_enhanced:
+            for item in daily_round.pain_scale_enhanced:
+                if 'scale' in item:
+                    item['scale'] /= 2
+                    item['scale'] = round(item['scale'])
```

Comment:
when we reverse `1`, it'd become `round(0.5) = 0`. This is below the minimum as per the schema

Reply:
maybe we could ceil instead of floor?

<img width="334" alt="image" src="https://github.com/coronasafe/care/assets/25143503/93a0a4b0-f694-486a-9cd1-00292e1f7137">